### DISCUSSION TITLE
How can I get the total number of pages ?

### DISCUSSION THREAD

**Discussion by ivaquero** [typst/typst#2720] [Q&A] [answered]
How can I get the total number of pages ?

**Comment by Dherse** [comment, marked as answer]
You can do counter(page).last(loc) where loc is the value returned by a locate call

**Reply by ivaquero** [reply]
Good idea. Thank you!

**Reply by Dherse** [reply]
Glad it works for you 🎉

**Reply by ivaquero** [reply]
By the way, on my laptop, the sentence is `counter(page).final(loc).first()` (typst v0.9.0).

**Comment by yochem** [comment]
This may be unrelated, but if you want to use this for page numbering, you can just use:'

```typst
#set page(numbering: "1/1")
```

**Reply by Munchotaur** [reply]
You can. But that doesn't work for something like "page 1/1".

**Comment by fee1-dead** [comment]
note that an updated answer for this would be
```typst
#context [
  #counter(page).final().at(0)
]
```
